### PR TITLE
Added New Swag Opportunity: Metafy

### PR DESCRIPTION
**Pull request by sshiv5768**

### CODE REVIEW

**Comment by sshiv5768** [comment]
@thefool76 have a look on this.

File: src/list.json [hunk 1]
```diff
@@ -203,6 +203,17 @@
         "description": "Meilisearch is once again back for Hacktoberfest! We look back on previous editions while getting ready for this year’s, armed with brand new common goals and plenty of exclusive new swag for our contributors.",
         "no_of_prs": 1,
         "link": "https://blog.meilisearch.com/contribute-hacktoberfest-2022/"
+      },
+      {
+        "organization": "Metafy",
+        "org_url": "https://github.com/metafy-social/.github/blob/main/profile/README.md",
+        "tags": [
+          "NFT",
+          "T-shirt"
```
Comment (thefool76) [inline src/list.json:212]:
```suggestion
          "T-shirt" => "Tshirt"
```

Comment (thefool76) [reply src/list.json:212]: Later on Please Re-style it using this command
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2340980/patch | git am`
`git push `

**Review by thefool76 — CHANGES_REQUESTED** [review]

File: src/list.json (hunk 1 above)
Comment (sshiv5768) [reply src/list.json:212]: Ok cool

**Pull request closed without merging**